src/main/kotlin/com/pubnub/api/subscribe/eventengine/event/Event.kt

```diff
@@ -24,13 +24,13 @@ sealed class Event : CoreEvent {
     data class HandshakeReconnectSuccess(val subscriptionCursor: SubscriptionCursor) : Event()
     data class HandshakeReconnectFailure(val reason: PubNubException) : Event()
     object HandshakeReconnectRetry : Event()
-    data class HandshakeReconnectGiveUp(val reason: PubNubException) : Event()
+    data class HandshakeReconnectGiveup(val reason: PubNubException) : Event()
```

Comment:
Why this change?

Reply:
This is how we named it in the spec https://github.com/pubnub/ce-decisions/blob/master/documents/Subscribe%20Event%20Engine%20Description.md#handshake_reconnect_giveup and it's how it's used in the contract tests https://github.com/pubnub/sdk-specifications/blob/001a288adefa3ac2ba1f02d83e5e8c955b0f4349/features/subscribe/event-engine/happy-path.feature#L47

Reply:
Ok, thank's for clarification.